apps/examples/testcase/le_tc/kernel/tc_task.c

```diff
@@ -541,6 +670,9 @@ int task_main(void)
 	tc_task_task_reparent();
 #endif
 	tc_task_task_init();
+#ifdef CONFIG_TASK_MONITOR
+	tc_task_task_monitor_utils();
```

Comment:
TC name can be like below.
`tc_<module>_<API_name>`
So it can be `tc_task_task_monitor_register`.